Dwarf.cpp

```diff
@@ -1,7 +1,23 @@
 #include "Dwarf.h"
+#include "Utility.h"
 
-//Dwarf::Dwarf
+Dwarf::Dwarf(std::string n,int hp, int armor) : Character(hp, armor, 4), name(n)
+{
+    helpfulItems = makeHelpfulItems(3);
+    defensiveItems = makeDefensiveItems(2);
+}
 
-//Dwarf::getName
+const std::string& Dwarf::getName()
+{
+    return name;
+}
 
-//Dwarf::getStats
\ No newline at end of file
+void Dwarf::attack(Character& other)
+{
+    Character::attack(other);
+}
```

Comment:
This function was not needed
it does nothing except call the base class's function
- [ ] remove it